Makefile

```diff
@@ -4,21 +4,29 @@ TEMP_DIR:=$(shell mktemp -d)
 ARCH?=amd64
 OUT_DIR?=./_output
 
+OPENAPI_PATH=$(GOPATH)/src/k8s.io/kube-openapi
+
 VERSION?=latest
 
+ifeq  ($(PLATFORM),Linux)
+SED_REPLACE :=
+else
+SED_REPLACE := ''
+endif
```

Comment:
Happy to split all of this out, this was done so the adapter is buildable on macos.